notebooks/06_Serving/06_02_kfserving_sdk.ipynb

```diff
@@ -0,0 +1,256 @@
+{
```

Comment:
Can we have a new section? 
1. Using YAML to deploy inferenceService
2. Using SDK to deploy the inferenceService

Reply:
It makes sense.